nexus/arch.cc

```diff
@@ -601,7 +601,12 @@ delay_t Arch::estimateDelay(WireId src, WireId dst) const
     int dst_x = dst.tile % chip_info->width, dst_y = dst.tile / chip_info->width;
     int dist_x = std::abs(src_x - dst_x);
     int dist_y = std::abs(src_y - dst_y);
-    return 75 * dist_x + 75 * dist_y + 250;
+
+    int mult = 75;
+    if (getCtx()->settings.count(getCtx()->id("estimate-delay-mult")))
```

Comment:
a call to ctx->id here is going to be adding an unnecessary string hash and compare to a hot path, can you make mult a field of arch that is loaded from settings before place/route instead?

Reply:
Right, thanks. I have added `mult` as a member of arch as suggested and the settings reading step just before placement.